lib/tapjoy/ldap/api/user.rb

```diff
@@ -0,0 +1,112 @@
+require 'memoist'
+module Tapjoy
+  module LDAP
+    module API
+      module User
+        class << self
+          extend Memoist
+          def create(fname, lname, type, group)
+            # Properly capitalize names
+            fname, lname = [fname, lname].map(&:titleize)
```

Comment:
I was thinking you could simply take the setup you have here:

`fname, lname = [fname, lname].map(&:titleize)`

and swap out `&:titleize` for `&:capitalize`